enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,197 @@
+  - <https://www.packet.com/developers/changelog/custom-image-handling/>
+- Packet does not offer a managed DNS solution. Route53, CloudFlare, or RFC 2136
+  (DNS Update), and NS record based zone forwarding are considerations. Existing
+  IPI drivers like BareMetal have similar limitations.
+- LoadBalancing can be achieved with the Packet CCM, which creates and maintains
+  a MetalLB deployment within the cluster.
+- Both Public and Private IPv4 and IPv6 addresses made available for each node.
+  Private and VLAN Networks can be restricted at the project level (default) or
+  node by node. While custom ASNs and BGP routing features are supported, these
+  and other networking features will not be exposed through the IPI.
+  <https://www.packet.com/cloud/network/>
+- Openshift with run directly on RHCOS directly on baremetal without need for a
+  VM layer.
+
+### Risks and Mitigations
+
+The official Packet ClusterAPI, CCM, and CSI drivers are all relatively new and
+may make undertake substantial design changes.  This project will either need to
+adopt and maintain the current releases, or adopt newer releases developed
+alongside this project.
+
+The CI will need to assure that Packet resources are cleanly released after failed and successful builds.  Packet is currently used in the baremetal provisioning tests, so there is some prior art available.  If the same account is used for testing the IPI, it may need account quota increases.
+
+## Design Details
+
+### Test Plan
+
+We will use existing UPI platforms, such as OpenStack, AWS, and GCP, as the
+inspiration for our testing strategy:
+
+- A new e2e job will be created
+- At the moment, we think unit tests will probably not be necessary. However, we
+  will cover any required changes to the existing codebase with appropriate
+  tests.
+
+### Graduation Criteria
+
+The proposal is to follow a graduation process based on the existence of a CI
+running suite with end-to-end jobs. We will evaluate its feedback along with
+feedback from QE's and testers.
+
+We consider the following as part of the necessary steps:
+
+- CI jobs present and regularly scheduled.
+- IPI document published in the OpenShift repo.
+- End to end jobs are stable and passing and evaluated with the same criteria as
+  comparable IPI providers.
+- Developers of the team have successfully used the IPI to deploy on Packet
+  following the documented procedure.
+
+## Implementation History
+
+Significant milestones in the life cycle of a proposal should be tracked in
+`Implementation History`.
+
+## Drawbacks
+
+The IPI implementation is provisioned on baremetal which faces resource
+availability limitations beyond those in VM environment. CI, QE, documentation,
+and tests will need to be generous when defining provisioning times. Tests and
+documentation should also be made flexible about facilities and device model
+choices to avoid physical limitations.
+
+## Alternatives
```

Comment:
Do we want to list  assisted-installer (AI) as another alternative here:
https://github.com/openshift/enhancements/blob/master/enhancements/installer/connected-assisted-installer.md

Reply:
as an alternative for doing installs on packet instead of a full IPI implementation, you mean?

Reply:
We could add to the open questions list something like "are any aspects of the assisted installer flow for bare metal clusters reusable and helpful for the Packet IPI install flow?"

I don't know the answer yet, but that would make sure we don't lose the question